File: app/src/main/java/com/example/rodrigo/sgame/CommonGame/Level.java
package com.example.rodrigo.sgame.CommonGame;


/**
 * Creator : Rodrigo Vidal Villaseñor
 */
public class Level {

    /**
     * Void Constructor
     */
    public Level(){}

    /**
     * Create a Level object for the adapter
     * @param numberLevel Number of level
     * @param gameType Type of step
     * @param tag Some additional information
     */
    public Level(int numberLevel, String gameType, String tag) {
        this.numberLevel = numberLevel;
        this.gameType = gameType;
        this.tag = tag;
    }

    public int getNumberLevel() {
        return numberLevel;
    }

    public void setNumberLevel(int numberLevel) {
        this.numberLevel = numberLevel;
    }

    public String getGameType() {
        return gameType;
    }

    public void setGameType(String gameType) {
        this.gameType = gameType;
    }

    public String getTag() {
        return tag;
    }

    public void setTag(String tag) {
        this.tag = tag;
    }

    int numberLevel;
    String gameType,tag;
}

File: app/src/main/java/com/example/rodrigo/sgame/CommonGame/Level.kt
package com.example.rodrigo.sgame.CommonGame

/**
 * Creator : Rodrigo Vidal Villaseñor
 */
class Level {
    var numberLevel: Int = 0
        internal set
    lateinit var gameType: String
        internal set
    lateinit var tag: String
        internal set

    /**
     * Void Constructor
     */
    constructor() {}

    /**
     * Create a Level object for the adapter
     * @param numberLevel Number of level
     * @param gameType Type of step
     * @param tag Some additional information
     */
    constructor(numberLevel: Int, gameType: String, tag: String) {
        this.numberLevel = numberLevel
        this.gameType = gameType
        this.tag = tag
    }
}
